tests/test_views.py

```diff
@@ -14,10 +14,12 @@
 from django.test.client import Client
 from django.urls import reverse
 
+from learning_assistant.models import LearningAssistantMessage
+
 User = get_user_model()
 
 
-class TestClient(Client):
+class FakeClient(Client):
```

Comment:
This class was renamed to remove the following test warning:
```
tests/test_views.py:22
  /Users/(ueser)/edx/src/learning-assistant/tests/test_views.py:22: PytestCollectionWarning: cannot collect test class 'TestClient' because it has a __init__ constructor (from: tests/test_views.py)
    class TestClient(Client):
```